instrumentation-genai/opentelemetry-instrumentation-vertexai/src/opentelemetry/instrumentation/vertexai/__init__.py

```diff
@@ -39,42 +39,66 @@
 ---
 """
 
+from __future__ import annotations
+
 from typing import Any, Collection
 
 from wrapt import (
-    wrap_function_wrapper,  # type: ignore[reportUnknownVariableType]
+    wrap_function_wrapper,  # pyright: ignore[reportUnknownVariableType]
```

Comment:
```suggestion
    wrap_function_wrapper,  # pyright: ignore[reportUnknownVariableType]
```

Can we keep type: instead of pyright:? I think it's more common

Reply:
Ya sorry I think this was before we upgraded pyright to the version that supports `type: ignore[foo]`